pyproject.toml

```diff
@@ -1,11 +1,83 @@
-[tool.isort]
-multi_line_output = 3
-include_trailing_comma = true
-force_grid_wrap = 0
-use_parentheses = true
-ensure_newline_before_comments = true
-line_length = 88
-skip_gitignore = true
+[tool.poetry]
+name = "clinica"
+version = "0.5.1"
```

Comment:
If version will be set here, it seems that there is no reason to keep `clinica/VERSION` (to verify).

Reply:
indeed, topic for a follow-up PR

Reply:
How is this version number updated?